settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:console_authorize_with_ssl_client: true
+:console_authorize_with_trusted_hosts: false
```

Comment:
Shorter names would be nice, but at the cost of being less self-explanatory. How about something like `console_ssl_auth` and `console_trusted_hosts_auth`?